pkg/clusteragent/admission/mutate/autoinstrumentation/config.go

```diff
@@ -304,16 +304,19 @@ func (n NamespaceSelector) AsLabelSelector() (labels.Selector, error) {
 // variables to the workload that matches targeting.
 type TracerConfig struct {
 	// Name is the name of the environment variable.
-	Name string `mapstructure:"name" json:"name"`
```

Comment:
Good call on this. These mapstructure tags were needed before we switched to the Datadog `structure` package.